locals.tf

```diff
@@ -0,0 +1,131 @@
+    production  = local.ucfs_london_bootstrap_servers[local.environment] // now on UCFS Production HA
+  }
+
+  kafka_broker_port = {
+    development = local.stub_kafka_broker_port_https
+    qa          = local.stub_kafka_broker_port_https
+    integration = local.k2hb_data_source_is_ucfs[local.environment] ? local.uc_kafka_broker_port_https : local.stub_kafka_broker_port_https
+    preprod     = local.uc_kafka_broker_port_https
+    production  = local.uc_kafka_broker_port_https
+  }
+
+  kafka_consumer_truststore_aliases = {
+    development = "ucfs_ca"
+    qa          = "ucfs_ca"
+    integration = "ucfs_ca"
+    preprod     = "ucfs_ca"
+    production  = "ucfs_ca,ucfs_ca_old"
+  }
+
+  kafka_consumer_truststore_certs = {
+    development = "s3://${local.certificate_auth_public_cert_bucket.id}/ca_certificates/ucfs/root_ca.pem"
+    qa          = "s3://${local.certificate_auth_public_cert_bucket.id}/ca_certificates/ucfs/root_ca.pem"
+    integration = "s3://${local.certificate_auth_public_cert_bucket.id}/ca_certificates/ucfs/root_ca.pem"
+    preprod     = "s3://${local.certificate_auth_public_cert_bucket.id}/ca_certificates/ucfs/root_ca.pem"
+    production  = "s3://${local.certificate_auth_public_cert_bucket.id}/ca_certificates/ucfs/root_ca.pem,s3://${local.certificate_auth_public_cert_bucket.id}/ca_certificates/ucfs/root_ca_old.pem"
+  }
+
+  log_level = {
+    development = "DEBUG"
+    qa          = "INFO"
+    integration = "INFO"
+    preprod     = "INFO"
+    production  = "INFO"
+  }
+
+  kafka_bootstrap_servers = join(
+    ",",
+    formatlist(
+      "%s:%s",
+      local.kafka_london_bootstrap_servers[local.environment],
+      local.kafka_broker_port[local.environment],
+    ),
+  )
+
+  kafka_consumer_group = "dataworks-ucfs-kafka-to-hbase-ingest-${local.environment}"
+
+  kafka_fetch_max_bytes = {
+    development = 20000000
+    qa          = 20000000
+    integration = 20000000
+    preprod     = 20000000
+    production  = 20000000
+  }
+
+  kafka_max_partition_fetch_bytes = {
+    development = 20000000
+    qa          = 20000000
+    integration = 20000000
+    preprod     = 20000000
+    production  = 20000000
+  }
+
+  kafka_max_poll_interval_ms = {
+    development = 600000
+    qa          = 600000
+    integration = 600000
+    preprod     = 600000
+    production  = 1800000
+  }
+
+  kafka_max_poll_records = {
+    development = 25
+    qa          = 50
+    integration = 50
+    preprod     = 25
+    production  = 5000
+  }
+
+  kafka_poll_duration_seconds = {
+    development = 10
+    qa          = 10
+    integration = 60
+    preprod     = 60
+    production  = 120
+  }
+
+  kafka_topic_regex = {
+    //match any "db.*" collections i.e. db.aa.bb, with only two literal dots allowed
+    //DW-4748 & DW-4827 - Allow extra dot in last matcher group for db.crypto.encryptedData.unencrypted
+    development = "^(db[.]{1}[-\\\\w]+[.]{1}[-.\\\\w]+)$"
```

Comment:
these arn't the right topics for this consumer

Reply:
Accepted that we are leaving these until we know the exact regex once connectivity is completed.